src/events/deposit.controller.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Controller, Get, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { ApiKeyGuard } from '../auth/guards/api-key.guard';
+import { DepositService } from './deposit.service';
+import { SerializedDeposit } from './dto/serialized-deposit.dto';
+
+@ApiTags('Deposit')
+@Controller('deposits')
+export class DepositController {
```

Comment:
nit for consistency
```suggestion
export class DepositsController {
```

Can the file name be made plural too for consistency?

Reply:
done

Reply:
yeah I missed that, it was sloppy of me. Thanks.